Packs/GooglePubSub/Integrations/GooglePubSub/README.md

```diff
@@ -0,0 +1,958 @@
+        },
+        {
+            "ackDeadlineSeconds": 10,
+            "expirationPolicy": {
+                "ttl": "86400s"
+            },
+            "labels": {
+                "test": "true"
+            },
+            "messageRetentionDuration": "86400s",
+            "name": "projects/dmst-integrations/subscriptions/test_sub_1587039587203",
+            "pushConfig": {},
+            "topic": "projects/dmst-integrations/topics/dmst-topic"
+        },
+        {
+            "ackDeadlineSeconds": 10,
+            "expirationPolicy": {
+                "ttl": "86400s"
+            },
+            "labels": {
+                "test": "true"
+            },
+            "messageRetentionDuration": "86400s",
+            "name": "projects/dmst-integrations/subscriptions/test_sub_1587040075117",
+            "pushConfig": {},
+            "topic": "projects/dmst-integrations/topics/dmst-topic"
+        },
+        {
+            "ackDeadlineSeconds": 10,
+            "expirationPolicy": {
+                "ttl": "86400s"
+            },
+            "labels": {
+                "test": "true"
+            },
+            "messageRetentionDuration": "86400s",
+            "name": "projects/dmst-integrations/subscriptions/test_sub_1587042146495",
+            "pushConfig": {},
+            "topic": "projects/dmst-integrations/topics/dmst-topic"
+        },
+        {
+            "ackDeadlineSeconds": 10,
+            "expirationPolicy": {
+                "ttl": "86400s"
+            },
+            "labels": {
+                "test": "true"
+            },
+            "messageRetentionDuration": "86400s",
+            "name": "projects/dmst-integrations/subscriptions/test_sub_1587043084505",
+            "pushConfig": {},
+            "topic": "projects/dmst-integrations/topics/dmst-topic"
+        },
+        {
+            "ackDeadlineSeconds": 10,
+            "expirationPolicy": {
+                "ttl": "86400s"
+            },
+            "labels": {
+                "doc": "true"
+            },
+            "messageRetentionDuration": "86400s",
+            "name": "projects/dmst-integrations/subscriptions/doc_sub",
+            "pushConfig": {},
+            "topic": "projects/dmst-integrations/topics/dmst-topic"
+        },
+        {
+            "ackDeadlineSeconds": 600,
+            "expirationPolicy": {},
+            "messageRetentionDuration": "604800s",
+            "name": "projects/dmst-integrations/subscriptions/gcf-function-1-us-central1-dmst-topic",
+            "pushConfig": {
+                "attributes": {
+                    "x-goog-version": "v1"
+                },
+                "pushEndpoint": "https://d4d1290519676f29baf13a7bf18a25bf-dot-j40fd5d18d8c290e1p-tp.appspot.com/_ah/push-handlers/pubsub/projects/dmst-integrations/topics/dmst-topic?pubsub_trigger=true"
+            },
+            "topic": "projects/dmst-integrations/topics/dmst-topic"
+        }
+    ]
+}
+```
+
+##### Human Readable Output
```

Comment:
not sure why this part is not marked blue. is that a GitHub issue?

Reply:
probably. The MD looks good inhttps://github.com/demisto/content/blob/8e3c54debe4eaf77526170709caf6c61a43b131c/Packs/GooglePubSub/Integrations/GooglePubSub/README.md